Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1510,9 +1510,16 @@ static void addUserInstalledFontURLs(NSString *path, HashMap<String, URL>& fontU
                 String fontName(fontNameAttribute.get());
                 String fontDisplayName(fontDisplayNameAttribute.get());
                 String fontFamilyName(fontFamilyNameAttribute.get());
-                fontURLs.add(fontName.convertToASCIILowercase(), fontURL);
-                fontURLs.add(fontFamilyName.convertToASCIILowercase(), fontURL);
-                didAddFontToMap = true;
+                auto lowerCaseFontName = fontName.convertToASCIILowercase();
+                if (!lowerCaseFontName.isEmpty()) {
```

Comment:
Is this API-testable?

Reply:
That is a good point! It requires an invalid user installed font, so it may not be trivial, but it's definitely doable. If time permits, I will look into creating an API test for this.

Thanks for reviewing!